src/arch/riscv/model/statedata.c

```diff
@@ -28,3 +29,6 @@
 asid_pool_t *riscvKSASIDTable[BIT(asidHighBits)];
 
 pte_t kernel_pageTables[CONFIG_PT_LEVELS][BIT(PT_INDEX_BITS)] ALIGN(BIT(seL4_PageTableBits));
+
+/* Address of pk trap address */
+UNUSED word_t pk_trap_addr;
```

Comment:
Not sure how `kernel_pageTables` above snuck through but the naming convention in statedata is `archKSStateName`, see `riscvKSASIDTable` further up and the definitions in all the other statedata